### PR TITLE
Prosthetic Limbs for Commissioned Officers

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Commissioned officers can now choose to spawn with prosthetic limbs. The following restrictions apply:

1) Only for commissioned officers. 2Lts and up can only have prosthetic legs. Captains and up can have prosthetic legs or arms.
2) You can only have one prosthetic.

## Why It's Good For The Game

It's lore-accurate, as Captain Jeremy Cruz of the USS Sephora had a prosthetic limb. It also opens an avenue for RP for commissioned officers that regular grunts do not get.

## Changelog

:cl:
add: Commissioned officers (CIC staff, IOs, DPs) can now have prosthetic limbs, selected via Character Traits in character setup.
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
civvies and survivors should be able to too

**Comment by Geevies** [comment]
> civvies and survivors should be able to too

out of scope

File: code/modules/character_traits/robotic_limbs.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		to_chat(target, SPAN_WARNING("You aren't a high enough rank to be eligible to have a prosthetic [robo_limb_name]!"))
+		return
+	var/obj/limb/limb = target.get_limb(code_limb)
+	limb.robotize()
+	target.update_body()
+	return ..()
+
+/datum/character_trait/robo_limb/left_leg
+	robo_limb_name = "Left Leg"
+	code_limb = "l_leg"
```
Comment (stanalbatross) [inline code/modules/character_traits/robotic_limbs.dm:32]: mfw these are not defines anywhere in code

Comment (Geevies) [reply code/modules/character_traits/robotic_limbs.dm:32]: ikr

**Pull request merged**